api/server/router/image/image_routes.go

```diff
@@ -236,12 +245,20 @@ func (s *imageRouter) postImagesLoad(ctx context.Context, w http.ResponseWriter,
 		return err
 	}
 	quiet := httputils.BoolValueOrDefault(r, "quiet", true)
+	name := r.FormValue("name")
+	var refs = map[string]string{}
+	refsJSON := r.FormValue("refs")
```

Comment:
Same here: the new options should be ignored on older API versions, and a default should be set (if needed) for older API versions

Reply:
:+1:

Reply:
already defaulted

Reply:
@runcom shouldn't they still be ignored on older API versions (and if experimental is disabled?)

Reply:
There's a != "" below this line which should make sure to ignore this

Reply:
Oh, what I meant was that if I do;

```
/v1.25/images/load?refs=something
```

It should ignore that option, even though it's set, because API v1.25 doesn't support it